gnovm/tests/files/zrealm_crossrealm24.gno

```diff
@@ -0,0 +1,30 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	crossrealm "gno.land/r/demo/tests/crossrealm"
+)
+
+// XXX, this equals to:
+//
+//	type LocalBar *crossrealm.Bar
+//	var b0 LocalBar
+//
+// both should be valid
+// TODO: expand this
+
+var b0 *crossrealm.Bar
+
+func init() {
+	b0 = &crossrealm.Bar{A: 1}
+}
+
+func main() {
+	print(".")
+}
+
+// XXX, should work to attach by reference to
+// type defined in another realm???
+
+// Output:
+// .
```

Comment:
is this right? `&crossrealm.Bar{A: 1}`(from external realm) is persisted in current realm as a heapItemValue: `heapitem((struct{(1 int)} gno.land/r/demo/tests/crossrealm.Bar))`

Reply:
I think this is right, from my interpretation. The pointer goes to a value in the other realm, but it is attached to r/demo/crossrealm, so it's ok?